cert/crypto.go

```diff
@@ -0,0 +1,82 @@
+package cert
+
+import (
+	"crypto/aes"
+	"crypto/cipher"
+	"crypto/rand"
+	"io"
+
+	"golang.org/x/crypto/argon2"
+)
+
+// Selected for AES-256
+const keySize = 32
+
+// KDF factors - roughly 250ms on my Macbook Pro
```

Comment:
I don't claim to be a crypto expert, so it's worth paying attention to these values here. Most reading online talks about KDFs in the context of web app user authentication, which isn't how it's used here. The general guidance is to "increase the values as far as possible" which isn't exactly the most specific guidance.

My thinking here was that this operation isn't generally performed from an actual Nebula node serving authentications - or that if it is, it's done irregularly. Still, 250ms may be overkill. Something like 50ms/100ms would definitely feel faster to a user and offer better performance if certs were being issued in bulk.

Furthermore, the count of threads may be "low" as far as servers are concerned, yet "high" as far as personal devices are concerned. The memory usage wouldn't work well in an embedded application - though I don't think that's really a target here. Just some food for thought.